CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed an issue where migration diagnostic tool would crash if the target actor's owner couldn't be found.
 - Fixed an issue where during shutdown unregistering NetGUIDs could cause an asset load and program stall.
 - Fix RPC timeouts for parameters referencing assets that can be asynchronously loaded.
+- Fixed the test settings overrides config filename in the `World Settings` so that the file path is relative to the game directory.
```

Comment:
```suggestion
- Fixed the test settings overrides config filename in `Spatial World Settings` so that the file path is relative to the game directory.
```